python/az/aro/azext_aro/_validators.py

```diff
@@ -10,7 +10,8 @@
 from azure.cli.core.commands.client_factory import get_mgmt_service_client, get_subscription_id
 from azure.cli.core.profiles import ResourceType
 from azure.cli.core.azclierror import CLIInternalError, \
-    InvalidArgumentValueError, RequiredArgumentMissingError
+    InvalidArgumentValueError, RequiredArgumentMissingError, \
```

Comment:
nit: we can use `import (...)` just as `python/az/aro/azext_aro/tests/latest/unit/test_validators.py`

Reply:
I made this change globally across our imports for consistency's sake.